### PR TITLE
Added script covering [UpdateDeviceList] sending to HMI 

### PR DESCRIPTION
**Pull request by aarotar**

### CODE REVIEW

**Comment by aarotar** [comment]
@vvvakulenko please review my pull request

**Comment by aarotar** [comment]
@aderiabin please review my pull request

File: test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+--        Connect device 
+
+-- Requirement summary: 
+--    [UpdateDeviceList] sending to HMI 
+--
+-- Expected result:
+--    SDL sends UpdateDeviceList to HMI right after new device connects over WiFi
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
```
Comment (vvvakulenko) [inline test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua]: @aarotar will connecttest_resumption.lua be commited separately? I don't see such file in user modules and can't run script

Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua]: @vvvakulenko  please use file from commit  	aece9d3

File: test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+					}
+	):Do(function(_,data)
+		self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+	end)
+	:Times(2)
+end
+
+--[[ Postconditions ]]
+commonFunctions:SDLForceStop()
+
```
Comment (aderiabin) [inline test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua]: Please, avoid blank lines at the end of script

Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua]: @aderiabin updated in commit  94fcc12

**Comment by aarotar** [comment]
@maria-mih please review

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**